RetrospectiveExtension.Frontend/components/actionItem.tsx

```diff
@@ -8,8 +8,9 @@ import {
   IDocumentCardPreviewProps
 } from 'office-ui-fabric-react/lib/DocumentCard';
 import * as React from 'react';
-import { WorkItem, WorkItemType, WorkItemStateColor } from 'TFS/WorkItemTracking/Contracts';
-import { WorkItemFormNavigationService } from 'TFS/WorkItemTracking/Services';
+import * as SDK from 'azure-devops-extension-sdk';
```

Comment:
do we need everything from SDK? if we can capture only the methods we're using, tree-shaker and code-trimmers might spit fewer bytes during the compile process.

Reply:
Let me check and fix other places as well.

Reply:
Seems to me that webpack is smart enough to only include what are needed so the bundle size doesn't change after I've fixed all of them. However, it's still a good practice to only include what you use in the file.